src/controller.coffee

```diff
@@ -214,6 +214,8 @@ exports.Editor = class Editor
     # ## Tracker Events
     # We allow binding to the tracker element.
     dispatchMouseEvent = (event) =>
+      # Firefox doens't have a global event object
+      window.event = event
```

Comment:
Yep, this is the long-term fix.  Should I do that fix now before check-in?  Adding this change for now will fix the user-facing bug in the meantime.